readme.txt

```diff
@@ -105,105 +105,87 @@ There are opportunities for developers at all levels to contribute. [Learn more
-* Infinite Scroll: the classes `infinite-scroll` and `neverending` are now applied using JS instead of PHP. #6346
-* Markdown: allow to independently toggle Markdown for posts. #6406
-* Photon: do not use Photon on images hosted on WordPress.com and already including resize parameters. #6497
-* Protect: update the support link appearing when you're locked out of your site. #6468
-* Publicize: add Unit Tests to make sure Publicize doesn't break when making changes to the Publicize process. #6018
-* Sharing: allow filtering of the shared link in the Email Sharing button. #6342
-* Recipe shortcode: update the library powering the Print link. #6263
-* Stats: add column in posts table with a link for each post to jump to its stats in WordPress.com. #6370
-* Google+ Shortcode: clean up enqueuing of the Google+ js library. #6440
-* Slideshow shortcode: add filter to customize speed of the slideshow. #3297
-* SlideShare shortcode: add filter to customize the output of the shortcode. #6440
-* Twitch.tv shortcode: update to use Twitch's new embedded player. #6338
-* Support: add more information to debug tools to help you get better help from our support team. #6171
-* Synchronization process: add filter to allow customizing sync timeout. #6317
-* Theme Tools: add option to use SVGs in Social Menus. #6096
-* VideoPress: add Video GUID to the media API endpoint. #6492
-* Widgets: add a filter to The Display Posts Widget content, so the content can be customized. #5883
-* Widgets: add a new email field to the Contact Info Widget. #6275
-* Widgets: if Photon is available, use it to serve the image in the Image and the Text widgets. #4220, #6434
-* Widgets: allow non-square image sizes in the Top Posts Widget. #6403
-* Widget Visibility: add ability to match ALL conditions. #6327
-* WordPress REST API: Use `register_rest_route()` consistently for registering JSON API endpoint routes. #6006
-* WordPress.com REST API: add date and time format settings to the Settings endpoint. #6343
-* WordPress.com REST API: add a filter for theme info results. #6446
-* WordPress.com REST API: allow Post backup method to be called with a site auth token. #6399
-* WordPress.com REST API: add new action links field to the plugins endpoint. #6267
-* WordPress.com REST API: add new endpoints to edit media via the API. #5281
+* We've added more information to our debug tools to improve your support experience.
+* New HTML5 versions of our house ads are out.
+* Update and display custom copyright EXIF information in the Carousel.
+* We've highlighted the ability to export Contact Form feedback as it was being overlooked.
+* If you have images on WordPress.com we're going to skip using Photon when the images include resize parameters.
+* Use the Sharing filter to customize the emails sent from the Email Sharing button.
+* We've updated the library powering the Print link.
+* Quickly jump to post specific stats on WordPress.com with a new link.

```

Comment:
This will be very noticeable as soon as folks update (it adds a new column with a blue stats icon to the Posts screen), so I think we should make it more noticeable in the changelog as well.